src/utils/utils.cpp

```diff
@@ -81,11 +94,14 @@ void tabs(std::ostream& os, int tabs, int spacesPerTab) {
     }
 }
 
-void print(std::stringstream& ss, Logging::Level lvl) {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-parameter"
+void print(std::stringstream& ss, Paper::LogLevel lvl) {
     ss << std::endl;
-    Logger::get().log(lvl, "%s", ss.str().c_str());
+    il2cpp_utils::getLogger().fmtLog<Paper::LogLevel::INF>("{}", ss.str().c_str());
```

Comment:
Is there no non-templated log by level here? If not, drop the `lvl` param and change the usage to not have the level specified, or move the entire definition to a header and make it templated.